src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -313,12 +425,32 @@ Digitized head points:
 
 Anatomical MRI:
 
+{{ MACROS___make_metadata_table(
+   {
+      "IntendedFor": (
+         "OPTIONAL",
+         "This is used to identify the structural MRI(s), "
+         "possibly of different types if a list is specified, "
+         "to be used with the MEG recording.",
+      )
+   }
+) }}
+
 | **Key name** | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                                                                                      |
 | ------------ | --------------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | IntendedFor  | OPTIONAL              | [string][] or [array][] of [strings][] | Path or list of path relative to the subject subfolder pointing to the structural MRI, possibly of different types if a list is specified, to be used with the MEG recording. The path(s) need(s) to use forward slashes instead of backward slashes (for example, `"ses-/anat/sub-01_T1w.nii.gz"`). |
 
 Anatomical landmarks:
 
+{{ MACROS___make_metadata_table(
+   {
+      "AnatomicalLandmarkCoordinates": "OPTIONAL",
+      "AnatomicalLandmarkCoordinateSystem": "OPTIONAL",
```

Comment:
in the [rendered docs](https://bids-specification--774.org.readthedocs.build/en/774/04-modality-specific-files/02-magnetoencephalography.html) this says:

> Preferably the same as the EEGCoordinateSystem.

However, given that this is the MEG spec, I think if should preferably be the same as MEGCoordinateSystem.

Reply:
I can append the appropriate "same as XXXCoordinateSystem" snippet to the end of each definition as a within-section modification, rather than a part of the central definition.

Reply:
Should be fixed in 670ca5b.